tools/util.wdl

```diff
@@ -83,20 +96,35 @@ task get_read_groups {
     >>>
 
     output {
-        File read_groups_file = "read_groups.txt"
+        File read_groups_file = "read_groups.txt"  # TODO change output to String type (or Array[String])
```

Comment:
Regarding the `TODO`, I would probably take a similar approach to other tools. We can generate a `File` output and a `Array[String]` output.

Reply:
I guess it's trivial to output the file as well, but what's the use case? I'd think we'd always prefer to work with the String directly, rather than need calls to `read_file()`. What scenario do you envision where having a file output is useful?

Reply:
I think the only purpose for the file would be storing the value as an output of the pipeline. I suppose it is easy enough for a use to dump the `@RG`s or look in the output `json`.

For the output, I'm fine with having a single `Array` object and in the STAR case, it just outputs a single entry. I think that's the easiest from a usability perspective.

Reply:
IMO I'd rather have this streamlined by removing the `File` output than supporting a potential edge-case.
1) for this task specifically, what pipeline would want read groups as an output?
2) as a general rule for our `String` tasks, I would discourage ever outputting a generic file like `read_groups.txt`. I think all our outputs should follow `prefix` conventions, and I'd rather not complicate this task by needing a `prefix` argument.

> I'm fine with having a single Array object and in the STAR case, it just outputs a single entry. I think that's the easiest from a usability perspective.

Fair enough. The "usability perspective" makes a strong case. Optional outputs are annoying. I'll make a quick commit addressing this.

Reply:
> 2\. as a general rule for our `String` tasks, I would discourage ever outputting a generic file like `read_groups.txt`. I think all our outputs should follow `prefix` conventions, and I'd rather not complicate this task by needing a `prefix` argument.

I didn't explain why I think this: When running batches, it's nice being able to aggregate all output files into the same directory. `prefix` conventions allow that to be a safe operation (unless you for some reason specify the same prefix in the same batch, which would be your fault IMO). Having any "generically named" files like `read_groups.txt` would cause clobbering, complicating the output process.